package-scripts/datadog-agent/prerm

```diff
@@ -11,14 +21,19 @@ if [ -f "/etc/debian_version" ] || [ "$LINUX_DISTRIBUTION" == "Debian" ] || [ "$
     else
         /etc/init.d/datadog-agent stop || true
     fi
+
+    remove_py_compiled_files
```

Comment:
So this is this going to be run only for the uninstall on debian? Because we're already doing it in the `preinst`